scss/jetpack-connect.scss

```diff
@@ -0,0 +1,28 @@
+// ==========================================================================
+// TOS Blurb
+// ==========================================================================
+
+.jp-jetpack-connect__tos-blurb {
+  font-size: .75rem;
+  margin-bottom: 2em;
+  text-align: left;
+
+  @media (min-width: 491px) {
+    text-align: center;
+  }
+}
+
+// ==========================================================================
+// iFrame
+// ==========================================================================
+
+.jp-jetpack-connect__iframe {
+  height: 240px;
+  width: 380px;
```

Comment:
Seems like we need varying height here and there. Should we have different classes for that?

Reply:
The iframe size is supposed to stay mostly fixed with D31768-code checked out. The only variation is for mobile vs. desktop (reflected in the media query below).

Reply:
Thanks for explaining 👍